### PR TITLE
Adiciona a opção de matar o inseto com spray

### PR DESCRIPTION
**Pull request by avlnx**
Adds another option to the portugues path

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2422)

### CODE REVIEW

**Comment by avlnx** [comment]
Check this out @casey-collab

File: portugues/spray/joaninha/liberar.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Meus parabéns! Sua misericórdia salvou sua vida e impediu a total destruição
```
Comment (casey-collab) [inline portugues/spray/joaninha/liberar.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: portugues/spray/joaninha/matar.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Oh não! Essa não era uma joaninha comum! Você acaba de exterminar a joaninha rainha
```
Comment (casey-collab) [inline portugues/spray/joaninha/matar.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by avlnx** [comment]
Check this out @casey-collab

**Comment by casey-collab** [comment]
Looks great! :fire:

**Pull request merged**